apps/tasks.py

```diff
@@ -166,14 +166,22 @@ def release_name(instance):
         rel_name.save()
 
 
-def post_delete_hooks(instance):
+def post_delete_hooks(appinstance):
     print("TASK - POST DELETE HOOK...")
```

Comment:
This print could be changed to the newly introduced logging I suppose.

Reply:
Yes, indeed!